include/ignition/gui/GuiEvents.hh

```diff
@@ -43,6 +43,158 @@ namespace ignition
         /// \brief Unique type for this event.
         static const QEvent::Type kType = QEvent::Type(QEvent::MaxUser);
       };
+
+      /// \brief The class for sending and receiving custom snap value events.
```

Comment:
I wonder if we can improve this documentation to provide more context. For a user who doesn't know about the snap functionality, the current documentation can be confusing.

Reply:
be5a482d8da3da972dff28a187723c805378d367